mne/utils/config.py

```diff
@@ -66,80 +66,90 @@ def set_memmap_min_size(memmap_min_size):
-    'MNE_KIT2FIFF_STIM_CHANNEL_CODING',
-    'MNE_KIT2FIFF_STIM_CHANNEL_SLOPE',
-    'MNE_KIT2FIFF_STIM_CHANNEL_THRESHOLD',
-    'MNE_LOGGING_LEVEL',
-    'MNE_MEMMAP_MIN_SIZE',
-    'MNE_REPR_HTML',
-    'MNE_SKIP_FTP_TESTS',
-    'MNE_SKIP_NETWORK_TESTS',
-    'MNE_SKIP_TESTING_DATASET_TESTS',
-    'MNE_STIM_CHANNEL',
-    'MNE_TQDM',
-    'MNE_USE_CUDA',
-    'MNE_USE_NUMBA',
-    'SUBJECTS_DIR',
-)
+known_config_types = {
+    'MNE_3D_OPTION_ANTIALIAS': '',
+    'MNE_3D_OPTION_DEPTH_PEELING': '',
+    'MNE_3D_OPTION_MULTI_SAMPLES': '',
+    'MNE_3D_OPTION_SMOOTH_SHADING': '',
+    'MNE_3D_OPTION_THEME': '',
+    'MNE_BROWSE_RAW_SIZE': (
+        'tuple, width and height of the raw browser window (in inches)'
+    ),
+    'MNE_BROWSER_BACKEND': '',
+    'MNE_BROWSER_OVERVIEW_MODE': '',
+    'MNE_BROWSER_PRECOMPUTE': '',
+    'MNE_BROWSER_THEME': '',
+    'MNE_BROWSER_USE_OPENGL': '',
+    'MNE_CACHE_DIR': '',
+    'MNE_COREG_ADVANCED_RENDERING': '',
+    'MNE_COREG_COPY_ANNOT': '',
+    'MNE_COREG_FULLSCREEN': '',
+    'MNE_COREG_GUESS_MRI_SUBJECT': '',
+    'MNE_COREG_HEAD_HIGH_RES': '',
+    'MNE_COREG_HEAD_OPACITY': '',
+    'MNE_COREG_HEAD_INSIDE': '',
+    'MNE_COREG_INTERACTION': '',
+    'MNE_COREG_MARK_INSIDE': '',
+    'MNE_COREG_PREPARE_BEM': '',
+    'MNE_COREG_ORIENT_TO_SURFACE': '',
+    'MNE_COREG_SCALE_LABELS': '',
+    'MNE_COREG_SCALE_BY_DISTANCE': '',
+    'MNE_COREG_SCENE_SCALE': '',
+    'MNE_COREG_WINDOW_HEIGHT': '',
+    'MNE_COREG_WINDOW_WIDTH': '',
+    'MNE_COREG_SUBJECTS_DIR': '',
+    'MNE_CUDA_DEVICE': '',
+    'MNE_CUDA_IGNORE_PRECISION': '',
+    'MNE_DATA': '',
+    'MNE_DATASETS_BRAINSTORM_PATH': '',
+    'MNE_DATASETS_EEGBCI_PATH': '',
+    'MNE_DATASETS_EPILEPSY_ECOG_PATH': '',
+    'MNE_DATASETS_HF_SEF_PATH': '',
+    'MNE_DATASETS_MEGSIM_PATH': '',
+    'MNE_DATASETS_MISC_PATH': '',
+    'MNE_DATASETS_MTRF_PATH': '',
+    'MNE_DATASETS_SAMPLE_PATH': '',
+    'MNE_DATASETS_SOMATO_PATH': '',
+    'MNE_DATASETS_MULTIMODAL_PATH': '',
+    'MNE_DATASETS_FNIRS_MOTOR_PATH': '',
+    'MNE_DATASETS_OPM_PATH': '',
+    'MNE_DATASETS_SPM_FACE_DATASETS_TESTS': '',
+    'MNE_DATASETS_SPM_FACE_PATH': '',
+    'MNE_DATASETS_TESTING_PATH': '',
+    'MNE_DATASETS_VISUAL_92_CATEGORIES_PATH': '',
+    'MNE_DATASETS_KILOWORD_PATH': '',
+    'MNE_DATASETS_FIELDTRIP_CMC_PATH': '',
+    'MNE_DATASETS_PHANTOM_4DBTI_PATH': '',
+    'MNE_DATASETS_LIMO_PATH': '',
+    'MNE_DATASETS_REFMEG_NOISE_PATH': '',
+    'MNE_DATASETS_SSVEP_PATH': '',
+    'MNE_DATASETS_ERP_CORE_PATH': '',
+    'MNE_DATASETS_EPILEPSY_ECOG_PATH': '',
+    'MNE_FORCE_SERIAL': '',
+    'MNE_KIT2FIFF_STIM_CHANNELS': '',
+    'MNE_KIT2FIFF_STIM_CHANNEL_CODING': '',
+    'MNE_KIT2FIFF_STIM_CHANNEL_SLOPE': '',
+    'MNE_KIT2FIFF_STIM_CHANNEL_THRESHOLD': '',
+    'MNE_LOGGING_LEVEL': '',
+    'MNE_MEMMAP_MIN_SIZE': '',
+    'MNE_REPR_HTML': (
+        'bool, represent some of our objects with rich HTML in a notebook '
+        'environment (default "true")'
```

Comment:
I'm unsure whether it makes sense to say what the default is here. Most of these config vars are simply missing by default, so "default" here means "what the code does if the config var is missing".  I worry that someone changing the code at some later date will not know/think to change the stated "default" here, and this information will become stale/incorrect.

Reply:
Yes, this is a valid concern. On the other hand, it might still be nice to know the default behavior. If I see these keys, I want to know the defaults. But this might need a bigger overhaul of our config system, so no need to implement everything here.

Reply:
```suggestion
        'environment'
```